### PR TITLE
Bug 1489859 Add Technology Preview notice to Init Containers

### PR DESCRIPTION
**Pull request by bmcelvee**
https://bugzilla.redhat.com/show_bug.cgi?id=1489859

### CODE REVIEW

File: architecture/core_concepts/containers_and_images.adoc [hunk 1]
```diff
@@ -45,6 +45,21 @@ endif::[]
 [[init-containers]]
 === Init Containers
 
+[IMPORTANT]
+====
+Init containers is a Technology Preview feature only.
```
Comment (ahardin-rh) [inline architecture/core_concepts/containers_and_images.adoc]: s/is/are

**Comment by ahardin-rh** [comment]
@bmcelvee Just one comment from me. Looks good! Thanks for tackling this. 💟

**Comment by bmcelvee** [comment]
Thanks, @ahardin-rh! @rjhowe PTAL

**Comment by ahardin-rh** [comment]
LGTM!

**Comment by bmcelvee** [comment]
[rev_history]
|xref:../architecture/core_concepts/containers_and_images.adoc#architecture-core-concepts-containers-and-images[Containers and Images]
|Added Technology Preview warning to the xref:../architecture/core_concepts/containers_and_images.adoc#init-containers[Init Containers] section.
%

**Pull request merged**